src/serac/physics/heat_transfer.hpp

```diff
@@ -337,7 +337,7 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
           auto source = source_function(x + p, ode_time_point_, u, du_dx, params...);
 
           // Return the source and the flux as a tuple
-          return serac::tuple{source * det(I_plus_dp_dX), serac::zero{}};
+          return serac::tuple{-1.0 * source * det(I_plus_dp_dX), serac::zero{}};
```

Comment:
I thought that the previous sign convention was opposite the one in solid mechanics, but I hadn't gotten the chance to double check it yet. Good catch.

Reply:
This is what we have in the theory docs, so I put it in here. I can go check the solid one.

Reply:
Confusingly, the solid docs have it the other way. So at least we're consistent with the documentation?